Formula/xq.rb

```diff
@@ -18,7 +18,16 @@ class Xq < Formula
   depends_on "go" => :build
 
   def install
-    system "go", "build", *std_go_args(ldflags: "-s -w -X main.version=#{version}")
+    ENV["CGO_ENABLED"] = "0"
+    build_date = Time.now.strftime("%FT%TZ")
+    ldflags = %W[
+      -s -w
+      -X main.commit=#{Utils.git_head}
+      -X main.version=#{version}
+      -X main.date=#{build_date}
+    ]
```

Comment:
```suggestion
    ldflags = %W[
      -s -w
      -X main.commit=#{Utils.git_head}
      -X main.version=#{version}
      -X main.date=#{time.rfc3339}
    ]
```

Reply:
time.rfc3339 looks like a reference to a commit date, not a build date. But let it be :)

Reply:
Yep, we prefer this for [build reproducibility](https://reproducible-builds.org/)